concepts/05 UI Components/DropDownBox/00 Getting Started with DropDownBox/25 Customize Appearance.md

```diff
@@ -0,0 +1,99 @@
+Specify the [height](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#width) properties to change the size of the DropDownBox text field. If you want to resize the drop-down field, use the [dropDownOptions](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#dropDownOptions) object.
+
+To specify one of the predefined styles for the DropDownBox text field, use the [stylingMode](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#stylingMode) property. To customize the field appearance, specify the [fieldTemplate](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#fieldTemplate).
```

Comment:
I know this is a century-old DevExtreme tradition, but I'll fight it wherever I can.

```suggestion
To specify one of the predefined styles for the DropDownBox text field, use the [stylingMode](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#stylingMode) property. To customize the field appearance, specify the [field template](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#fieldTemplate).
```